### PR TITLE
Fixed issue where the distance to in view was always saying that it was in the view

### PR DESCRIPTION
**Pull request by shichongrui**
So before, Math.max was getting an extra 0 as a parameter which would always win and then minus the variance from that would always result in the distanceFromTrigger to always be less than 0.  So I removed the 0. 

There was also another bug where if an img is not in the view yet, the Math.max call would return a negative number.  Then you subtract the variance from there and it's always a negative number.  So this is backwards from the desired functionality which is why I added the -1 \* there.

### CODE REVIEW

File: dist/postpone.polyfill.js [hunk 1]
```diff
@@ -39,7 +39,7 @@
       deltaVPT = window.scrollY - (el.offsetTop + el.offsetHeight);
       deltaVPL = window.scrollX - (el.offsetLeft + el.offsetWidth);
       deltaVPR = el.offsetLeft - (window.scrollX + window.innerWidth);
-      distanceFromInView = Math.max(deltaVPT, deltaVPB, deltaVPL, deltaVPR, 0);
+      distanceFromInView = -1 * Math.max(deltaVPT, deltaVPB, deltaVPL, deltaVPR);
```
Comment (uniqname) [inline dist/postpone.polyfill.js]: Why would this issue not manifest itself on the repo's demo page?

Comment (shichongrui) [inline dist/postpone.polyfill.js]: It does.  If you open up the demo and then inspect all of the images on the page, the images at the bottom have already had their data-postpone-src value placed into the src attribute even if it has not met the postpone threshold yet.  In the below image, only the first img tag should be loaded, but all of them are.

image

Comment (uniqname) [inline dist/postpone.polyfill.js]: It no do so 4 me.
screen shot 2014-10-31 at 4 36 06 pm

Comment (uniqname) [inline dist/postpone.polyfill.js]: Also for reference,
http://codepen.io/uniqname/pen/mCxks?editors=001

Comment (shichongrui) [inline dist/postpone.polyfill.js]: Strangeness.  I'm not sure why I was having issues with it.  I pulled down the code again and tried it out and didn't have any issues.  Anyways you can close this  if you want.

**Pull request closed without merging**